### PR TITLE
Update 03-large-files.md

### PR DESCRIPTION
**Pull request by sureshpantheor**
Closes #

## Summary

**[Doc Page Title](https://docs.pantheon.io/doc-title)** - <Enter a one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.>

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/guides/filesystem/03-large-files.md [hunk 1]
```diff
@@ -37,7 +42,7 @@ A code repository larger than 2GB increases the possibility of Git errors when c
 
 ## Large Files
 
-The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure is not optimized to store and deliver large files.
+The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure are not optimized to store and deliver large files.
```
Comment (djschaap) [inline source/content/guides/filesystem/03-large-files.md:45]:
```suggestion
The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure are not optimized to store and deliver large files.  The maximum size for any single file is 256 MiB.
```
Documenting the hard limit here may be redundant to the table below, but IMO it's better to call this out clearly (& repeatedly? :P ).

File: source/content/guides/filesystem/03-large-files.md [hunk 2]
```diff
@@ -49,13 +54,13 @@ The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure is
 |:----------------|--------------------------------------|-------------------------------------|
 | ≤ 100MiB        | <span  style="color:green">✔</span>  | Can be uploaded via any means       |
 | 100MiB - 256MiB | <span  style="color:orange">✔</span> | Must be uploaded over SFTP or rsync |
-| > 256MiB        | <span  style="color:red">❌</span>    | Must be hosted via 3rd-party CDN    |
+| > 256MiB        | <span  style="color:red">❌</span>    | Must be offloaded to an external filesystem|
```
Comment (djschaap) [inline source/content/guides/filesystem/03-large-files.md:57]:
```suggestion
| > 256MiB        | <span  style="color:red">❌</span>    | Must be offloaded to an external filesystem
```
(minor) We aren't terminating other rows with `|`; recommend staying consistent.

File: source/content/guides/filesystem/03-large-files.md [hunk 3]
```diff
@@ -49,13 +54,13 @@ The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure is
...
 
 ### CDNs
 
-We recommend using a CDN like Amazon S3 as a cost-effective file serving solution that allows uploads directly to S3 from your site without using Pantheon as an intermediary if you are distributing large binaries or hosting big media files.
+For hosting either large files or a very large number of  media files, we recommend external file systems like Amazon S3 or Google Cloud Storage. These offer a cost-effective solution and can allow direct uploads to an external service without utilizing the Pantheon Filesystem..
```
Comment (djschaap) [inline source/content/guides/filesystem/03-large-files.md:61]:
```suggestion
For hosting either large files or a very large number of media files, we recommend external file systems like Amazon S3 or Google Cloud Storage. These offer a cost-effective solution and can allow direct uploads without utilizing the Pantheon Filesystem.
```

File: source/content/guides/filesystem/03-large-files.md [hunk 4]
```diff
@@ -49,13 +54,13 @@ The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure is
...
 
-- Drupal sites can use a module such as [S3 File System](https://www.drupal.org/project/s3fs).
+- Drupal sites can use a modules such as [S3 File System](https://www.drupal.org/project/s3fs).
```
Comment (djschaap) [inline source/content/guides/filesystem/03-large-files.md:63]:
```suggestion
- Drupal sites can use a module such as [S3 File System](https://www.drupal.org/project/s3fs).
```

File: source/content/guides/filesystem/03-large-files.md [hunk 5]
```diff
@@ -64,17 +69,15 @@ You can also refer to our documentation for [Drupal](/drupal-s3) and [WordPress]
...
 ### Media and Email
 
 Refer to the [Media and Email](/guides/platform-considerations/media-email-support) section of the [Platform Considerations](/guides/platform-considerations) guide for more information on support for large media and email files.
 
 ## Large (>100GiB) File Backups
 
-Large backups take longer, use more resources, and have a higher likelihood of failing. A 100GiB compressed tarball is not a convenient solution. Sites with footprints over 200GiB or two million files cannot be backed up for this reason (although code and database are backed up as normal).
-
-Despite the lack of backups, file content is highly durable and stored on multiple servers.
+Large backups take longer, use more resources, and have a higher likelihood of failing. Sites exceeding 200GiB or containing over two million files can't be backed up (though code and databases are backed up as normal). Even without backups, file content is reliably stored across multiple servers.
```
Comment (djschaap) [inline source/content/guides/filesystem/03-large-files.md:80]:
```suggestion
Large backups take longer, use more resources, and have a higher likelihood of failing.
Filesystems exceeding 200 GiB or containing over two million files cannot be backed up (though code and databases are backed up as normal).
Even without filesystem backups, file content is protected by redundant storage.
```
"multiple servers" feels archaic in a cloud environment - how about using "redundant storage" instead?

File: source/content/guides/filesystem/03-large-files.md [hunk 6]
```diff
@@ -20,13 +20,18 @@ This section provides information on highly populated directories and large file
...
+To optimize performance:
 
-Refer to the [File (field) Paths](https://www.drupal.org/project/filefield_paths) module to help resolve issues with highly populated directories in Drupal, or browse [WordPress plugins](https://wordpress.org/plugins/) for other solutions.
+- If possible, restructure directories that contain more than 50,000 files.
+- Consider offloading files to dedicated external storage solutions, such as Amazon S3 or Google Cloud Storage.
+- Use content management settings in both WordPress and Drupal to organize uploaded content into directories based on the YYYY/MM or YYYY/MM/DD format or specific user criteria. This approach is preferable over consolidating all uploads into a single directory. For sites already facing this issue, the solution typically involves reorganizing the files into multiple directories and then updating the respective file tables in Drupal or WordPress.
```
Comment (djschaap) [inline source/content/guides/filesystem/03-large-files.md:32]:
```suggestion
- Use content management settings in both WordPress and Drupal to organize uploaded content into directories based on the YYYY/MM or YYYY/MM/DD format or specific user criteria. This approach is preferable over writing all uploads to a single directory. For sites already facing this issue, the solution typically involves reorganizing the files into multiple directories and then updating the respective file tables in Drupal or WordPress.
```

File: source/content/guides/filesystem/03-large-files.md [hunk 7]
```diff
@@ -20,13 +20,18 @@ This section provides information on highly populated directories and large file
...
+
+Useful tools for offloading are detailed below under CDNs
```
Comment (djschaap) [inline source/content/guides/filesystem/03-large-files.md:34]:
```suggestion
Useful tools for offloading are detailed below under CDNs.
```

**Review by djschaap — CHANGES_REQUESTED** [review]
Some editing suggestions but looks great overall.

**Comment by sureshpantheor** [comment]
Doug,
Let's go with the changes you have suggested

**Review by djschaap — APPROVED** [review]

**Pull request merged**